policyengine_us/parameters/gov/states/id/tax/income/main/head_of_household.yaml

```diff
@@ -12,30 +12,37 @@ metadata:
       href: https://tax.idaho.gov/taxes/income-tax/individual-income/individual-income-tax-rate-schedule/
     - title: IDAHO STATUTES || Section 35.01.01.075 - TAX ON INDIVIDUALS, ESTATES, AND TRUSTS   
       href: https://casetext.com/regulation/idaho-administrative-code/title-idapa-35-tax-commission-state/rule-350101-income-tax-administrative-rules/section-350101075-tax-on-individuals-estates-and-trusts
-    - title: IDAHO STATUTES, TITLE 63, CHAPTER 30, SECTION 63-3024
-      href: https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3024/ 
     - title: LEGISLATIVE SESSIONS || 2022 EXTRAORDINARY LEGISLATIVE SESSION || 2022 EXTRAORDINARY SESSION LEGISLATIONHOUSE BILL 1
       href: https://legislature.idaho.gov/sessioninfo/2022spcl/legislation/H0001/
+    - title: Idaho state tax commission 2023 tax update
+      href: https://tax.idaho.gov/taxes/tax-pros/tax-update/tax-update-summer-2023/#:~:text=Idaho's%20tax%20rate%20is%20now,if%20you're%20filing%20jointly
+    - title: IDAHO STATUTES, TITLE 63, CHAPTER 30, SECTION 63-3024 (2)(a)
+      href: https://legislature.idaho.gov/statutesrules/idstat/title63/t63ch30/sect63-3024/ 
```

Comment:
the law does not seem to align with our source, it states 5.8% over 2,500 or 5,000 

Which one is correct?

Reply:
I think it should be 2,500 and 5,000 but please double check, that is what the law states